microsoft-edge/extensions-chromium/publish/contact-extensions-team.md

```diff
@@ -50,25 +62,32 @@ If you have any issues while you submit your extension to Partner Center, see [P
 
 
 <!-- ====================================================================== -->
-## Feature your extension in a collection
+## Feature an extension in a collection
 
 To feature your extension in a collection on the Microsoft Edge Add-ons website, see [Submit a request to add an extension to the collections on the Microsoft Edge Add-ons home page](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbRw01UwyBfAxNna_1ZkP3X2VUN0lBSU1YMEU3VFY0VURRODEwSjgwU00yRy4u) and complete the form.
 
 
 <!-- ====================================================================== -->
-## MicrosoftEdge-Extensions discussion forum
+## Microsoft Edge Insider Forum at Tech Community
+<!-- not specific to extensions -->
 
-Participate in the [Discussions](https://github.com/microsoft/MicrosoftEdge-Extensions/discussions) forum in the MicrosoftEdge-Extensions repo to flag any issues, raise a feature request, be a part of discussions, and share your ideas regarding the Microsoft Edge Extensions Store.
+For discussion about Microsoft Edge, including Beta, Dev, and Canary preview channels, use the [Microsoft Edge Insider](https://techcommunity.microsoft.com/t5/microsoft-edge-insider/ct-p/MicrosoftEdgeInsider) forum at Microsoft Tech Community.
 
 
 <!-- ====================================================================== -->
-## Report copyright or trademark infringement
+## Microsoft Edge team on X (Twitter)
+<!-- not specific to extensions -->
 
-If you think an item in the Microsoft Edge Add-ons website violates a copyright or trademark, complete the [Reporting Infringement](https://www.microsoft.com/concern/dmca) form.  When filling in **Step 3: Where can the infringing material be found?**, in the **Product/Service/App** option, make sure to select **Microsoft Store on Windows**.  The Microsoft Edge extensions team will review your report and then take the necessary action.
+For news and updates about developing for the web with Microsoft Edge, follow the [Microsoft Edge Dev (@MSEdgeDev)](https://twitter.com/msedgedev/) team on X (Twitter).
 
 
 <!-- ====================================================================== -->
-## See also
+## Learn about correct use of APIs
```

Comment:
I'm deleting this h2 section "Learn about correct use of APIs", which is the least relevant section in this .md.  I'm checking the two linked articles, doubt either article has anything to do with contacting - confirmed.  Note conflicting link text "Port Chromium".
@captainbrosset 
https://learn.microsoft.com/microsoft-edge/extensions-chromium/developer-guide/api-support
https://learn.microsoft.com/microsoft-edge/extensions-chromium/developer-guide/port-chrome-extension - has a Contact link, but the article is not about contacting.